pkg/test/oci_layout.go

```diff
@@ -550,3 +556,27 @@ func (olu BaseOciLayoutUtils) GetExpandedRepoInfo(repoName string) (RepoInfo, er
 
 	return repo, nil
 }
+
+func (olu BaseOciLayoutUtils) ExtractImageDetails(
+	repo, tag string, //nolint:unparam // function only called in the tests
```

Comment:
This nolint comment should not be needed.

Reply:
Comment removed